examples/tutorials/text.py

```diff
@@ -0,0 +1,128 @@
+"""
+Plotting text
+=============
+
+It is often useful to add annotations to a map plot. This is handled by
+:meth:`pygmt.Figure.text`.
+"""
+
+import pygmt
+
+########################################################################################
+# Basic map annotation
+# --------------------
+#
+# Text annotations can be added to a map using the :meth:`text` method of the
+# :class:`pygmt.Figure`.
+#
+# Full details of the GMT6 command `text` can be found `here<https://docs.generic-mapping-tools.org/latest/text.html>`_.
+# The Python binding to this command is documented `here<https://www.pygmt.org/latest/api/generated/pygmt.Figure.text.html>`_.
+#
+# Here we create a simple map and add an annotation using the ``text``, ``x``,
+# and ``y`` arguments to specify the annotation text and position in the
+# projection frame. ``text``, ``x``, and ``y`` accept `int`, `str`, or `float`.
```

Comment:
I'm not sure if it's a good idea to accept *x* and *y* in string-type.

Reply:
Should I remove that part of the line, pending possibly disallowing strings to be passed in?

Reply:
Yes.

Reply:
```suggestion
# projection frame. ``text``, ``x``, and ``y`` accepts either `int` or `float`.
```